FlowCrypt/Common/Imap.swift

```diff
@@ -368,3 +314,19 @@ enum MailDestination {
         }
     }
 }
+
+
+enum FCError: Error {
```

Comment:
Why FC Error in IMAP file? I suggest somewhere more general, like original error file.